src/interface/src/app/standalone/planning-areas/query-params.service.ts

```diff
@@ -16,7 +16,7 @@ export interface QueryParams extends Partial<Sort> {
 
 @Injectable()
 export class QueryParamsService {
-  readonly defaultLimit = 12;
+  readonly defaultLimit = 10;
```

Comment:
For the future, I wonder if this should be an imported const or even a user setting (I already forgot that we have a user prefs table) but this is fine for now.

Reply:
an option i was considering is doing something similar to what we are doing with the sort options

```
// Injection token used as default sorting options for this service
export const DEFAULT_SORT_OPTIONS = new InjectionToken<Sort>(
  'DEFAULT_SORT_OPTIONS'
);
```

and let the parent component handle these via injection token. What do you think?